src/Traits/Crudable/SearchManagement.php

```diff
@@ -31,7 +31,7 @@ private function eloquentSearch()
             $query->orderBy(request()->get('sort_by', 'id'), request()->get('sort_direction', 'desc'));
         }
 
-        if (!isset($this->pagination) || !$this->pagination || request()->get('pagination') === 'false') {
+        if ((isset($this->pagination) && !$this->pagination ) || request()->get('pagination') === 'false') {
```

Comment:
Ale tu to się nie zgodzę a co z przypadkiem że nie ma paginacji? Przecież to defaultowy case !isset($this->pagination)

Reply:
defaultowo ma być paginacja czy ma jej nie być ? - bo z komentarza w CR wywnioskowałem że jak nie ma w ogóle "pagination" w controlerze - to jest paginacja domyslna.
Dopiero jak celowo nie chcesz paginacji to ustawiasz $pagination=false, lub w requeście idzie "false" 
- jeśli tak ma byc to wszystko zadziała...bo ten warunek modyfikowany daje wyniki BEZ paginacji...